src/components/admin/app-navbar/AppNavbar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+
+  <vuestic-navbar>
+    <header-selector slot="selector" v-model="valueProxy"/>
+    <span slot="logo" class="i-vuestic"></span>
+    <span slot="center">
+      {{$t('navbar.messageUs')}}&nbsp;<a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
+    </span>
+    <message-dropdown :options="this.messages"></message-dropdown>
```

Comment:
You probably want to keep these options in corresponding components. They're not reusable in any way.

Also, don't use `this` in template.